src/views/helpers/index.ts

```diff
@@ -403,6 +403,14 @@ const NEW_VIEW_CREATORS = {
           colorField: '_value',
           colors: DEFAULT_THRESHOLDS_GEO_COLORS,
           isClustered: false,
+          tooltipColumns: [
```

Comment:
I don't think every dataset provided will have these columns. A lot of the times we filter out the columns and the column names like `s2_cell_id` can be named differently. What happens if this array is empty as it should be unless populated right?